### PR TITLE
First attempt at installing Toggle on recore

### PR DESCRIPTION
**Pull request by eliasbakken**

### CODE REVIEW

File: roles/install_toggle/meta/main.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 dependencies:
-  - { role: install_octoprint }
-  - { role: install_framebuffer }
\ No newline at end of file
+  - { role: install_octoprint, when: platform != 'recore' }
```
Comment (goeland86) [inline roles/install_toggle/meta/main.yml:3]: Why does Toggle not depend on OctoPrint when you're not installing on Recore?

File: roles/install_weston/tasks/recore.yml [hunk 2]
```diff
@@ -45,8 +45,5 @@
       WantedBy=multi-user.target
 
 - name: Enable service weston
-  tags: weston
-  service:
-    name: weston
-    enabled: yes
+  shell: systemctl enable weston
```
Comment (goeland86) [inline roles/install_weston/tasks/recore.yml:48]: There's a `systemd` module in Ansible you could use here instead: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html

File: roles/install_toggle/meta/main.yml (hunk 1 above)
Comment (eliasbakken) [reply roles/install_toggle/meta/main.yml:3]: It does, but it took forever to run the script with running through the whole octoprint installation first. When running the SYSTEM_klipper_octoprint-DEFAULT.yml the roles are all defined there, so we are not really using the dependencies. I think they should be removed for all platforms really.

File: roles/install_weston/tasks/recore.yml (hunk 2 above)
Comment (eliasbakken) [reply roles/install_weston/tasks/recore.yml:48]: I was getting an error when using the systemd module when running the INSTALL-toggle.yml script. It seems to work well when running using the build-* command.

**Pull request merged**